src/app/common/utils/system-notifications.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
```

Comment:
You should place this file and the test for it in `system-notifications` directory because later we can have more utils besides this one, it will be a mess if we will place everything in one directory. Also you should move to it and rename file `domains.wallets.enums.ts`. This notifications are not wallet's related but will be used in the whole application.